### PR TITLE
4827 Create Database

### PR DESCRIPTION
**Pull request by iamPhong**
https://edu-redmine.framgia.vn/issues/4827

### CODE REVIEW

**Comment by iamPhong** [comment]
Ready

File: app/assets/stylesheets/scaffolds.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
+body {
+  background-color: #fff;
```
Comment (mdmahmudurrahman) [inline app/assets/stylesheets/scaffolds.scss]: Declare the color variable in an external file, than import it here.

File: app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  color: #333;
```
Comment (mdmahmudurrahman) [inline app/assets/stylesheets/scaffolds.scss]: The same here.

File: app/assets/stylesheets/scaffolds.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+p, ol, ul, td {
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
+  line-height: 18px;
+}
+
+pre {
+  background-color: #eee;
```
Comment (mdmahmudurrahman) [inline app/assets/stylesheets/scaffolds.scss]: Same as above

File: app/controllers/chatrooms_controller.rb [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    end
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
+    def set_chatroom
+      @chatroom = Chatroom.find(params[:id])
+    end
+
+    # Never trust parameters from the scary internet, only allow the white list through.
```
Comment (mdmahmudurrahman) [inline app/controllers/chatrooms_controller.rb]: Please remove all of the comments.

File: app/controllers/chatrooms_controller.rb [hunk 5]
```diff
@@ -0,0 +1,74 @@
+class ChatroomsController < ApplicationController
```
Comment (mdmahmudurrahman) [inline app/controllers/chatrooms_controller.rb]: 1. Commit message is about "Create database". Why you created the controller 
2. Are you using all of the actions in this pull request? Please remove the actions that are not using. 
3. Remove all of the comments. 
4. Remove the scaffold-ed views that are not related to this pull request.

Comment (iamPhong) [reply app/controllers/chatrooms_controller.rb]: I'm using code of pull request task "init project", so more than 15 file changed. 
The code must have 'init project', then the application can run @@

Comment (iamPhong) [reply app/controllers/chatrooms_controller.rb]: I'm using command 'scaffold' , so it created controller

**Comment by iamPhong** [comment]
Ready

**Comment by mdmahmudurrahman** [comment]
commented 
Please don't scaffold, create necessary actions and views manually when you need it. :D

File: app/controllers/chatrooms_controller.rb (hunk 5 above)
Comment (iamPhong) [reply app/controllers/chatrooms_controller.rb]: Ok. I'm fine @mdmahmudurrahman

Comment (iamPhong) [reply app/controllers/chatrooms_controller.rb]: I'm removed file 'scaffold' created

File: app/controllers/chatrooms_controller.rb [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  before_action :set_chatroom, only: [:show, :edit, :update, :destroy]
```
[the remark this answers is not in the document]
Comment (iamPhong) [reply app/controllers/chatrooms_controller.rb]: I got it :D

**Comment by iamPhong** [comment]
Ready

File: app/controllers/chatrooms_controller.rb (hunk 6 above)
[the remark this answers is not in the document]
Comment (iamPhong) [reply app/controllers/chatrooms_controller.rb]: I'm remote this file. Next time, I will fix it

**Comment by mdmahmudurrahman** [comment]
commented
Fix conflicts

**Comment by iamPhong** [comment]
ready

**Comment by mdmahmudurrahman** [comment]
commented
Still have conflicts.

**Comment by iamPhong** [comment]
Ready

**Comment by mdmahmudurrahman** [comment]
commented
Please change 5 commits to 1 commit(1 commit per pull)

**Comment by iamPhong** [comment]
Ready

File: .gitignore [hunk 7]
```diff
@@ -47,3 +50,4 @@ bower.json
 # Ignore node_modules
 node_modules/
 
+test/*
```
Comment (huongnguyenmta) [inline .gitignore]: duplicate define with line 14

File: app/helpers/chatrooms_helper.rb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+module ChatroomsHelper
```
Comment (huongnguyenmta) [inline app/helpers/chatrooms_helper.rb]: remove all of files unused in this pull request

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>ROe13Chatroom</title>
```
Comment (huongnguyenmta) [inline app/views/layouts/application.html.erb]: I18n

File: config/credentials.yml.enc [hunk 10]
```diff
@@ -0,0 +1 @@
+P5wtPbv+No9AIMscQXjaIteNiIgmujTqdrNF69Ywnqkq+qkaK/s5LO6T0q3YZ0gESj2hadQum4R55ZxcU/WEgIdvARmpKTtQxV6bie1k/XsrWf4Xx9qXifjsmS9ISStyn7kGqmrUS9mXfqIqa9bCNAWky8OH+uvgYkPmgCJBtDfqaRqyk5EK3PkgBqCxBZnkJS6DGV+tIQMCNaWwp9o3MvJRxicy+IPq70JApd5PRAzzploHrEL4LZeREX1J+ACxYP6Q578jfYMDsqzMqofbCXNXngkyXYxkpoL6QR/tGWI+YdOtmwGcBVLHnEErBauwilpXF7RoR9wfsG4XASxZ5gqfumExJ/qG1al73q6Hxn2/GkuWzHS4KoQyj6mvkY83gnWrRM0tMGQbUUQMbEJRwqVvVFikHq7z7w0z--3JiyTv0CAXpqH/sz--n2HOmMnx1LHenzeDuNgncA==
```
Comment (huongnguyenmta) [inline config/credentials.yml.enc]: remove git cache this file

**Comment by huongnguyenmta** [comment]
commented
pull có conflicted em nhé
chị có thêm comment nữa =))
